spk/owncloud/src/wizard_templates/install_uifile.sh.yml

```diff
@@ -0,0 +1,14 @@
+OWNCLOUD_ADMIN_CONFIGURATION_STEP_TITLE: "ownCloud admin configuration"
```

Comment:
I believe this file should be named without the .sh part.
Sorry for not documenting that !
I will update the doc today.

Reply:
Yes, that did the trick. Much thanks!